src/components/CardList/card-list.jsx

```diff
@@ -54,19 +54,29 @@ export class CardList extends Component {
   }
 
   /**
-   * Take a suggestion from a CDS service based on action on from a card. Also pings the analytics endpoint (if any) of the
-   * CDS service to notify that a suggestion was taken
+   * Take a suggestion from a CDS service based on action on from a card. Also pings the feedback
+   * endpoint of the CDS service to notify that a suggestion was taken.
    * @param {*} suggestion - CDS service-defined suggestion to take based on CDS Hooks specification
-   * @param {*} url - CDS service endpoint URL
+   * @param {*} cardUUID - UUID of the card containing the suggestion
+   * @param {*} serviceUrl - CDS service endpoint URL
    */
-  takeSuggestion(suggestion, url) {
+  takeSuggestion(suggestion, cardUUID, serviceUrl) {
     if (!this.props.isDemoCard) {
       if (suggestion.label) {
-        if (suggestion.uuid) {
+        if (suggestion.uuid && cardUUID) {
           axios({
             method: 'POST',
-            url: `${url}/analytics/${suggestion.uuid}`,
-            data: {},
+            url: `${serviceUrl}/feedback`,
+            data: {
```

Comment:
Based on this PR https://github.com/cds-hooks/docs/pull/519/files#diff-e6669c257204becd7f07cdba801c0fe6R662-R671, the structure contains a `feedback` key and an array of feedback objects.

Reply:
@kolkheang you're absolutely right.  I was focused on the current card and glossed over the endpoint accepting feedback for multiple cards at once.

I've uploaded a new revision with the following example output:

```json
{
   "feedback":[
      {
         "card":"d9a6150a-c428-4076-965b-e839bf784028",
         "outcome":"accepted",
         "acceptedSuggestions":[
            {
               "id":"ac1d5585-2c50-4f15-b8f6-d0cd8d90d6ad"
            }
         ],
         "outcomeTimestamp":"2020-05-06T21:20:11.319Z"
      }
   ]
}
```